### PR TITLE
Gives some transparency to the bottom half of left/right-facing doors when open.

### LINKED ISSUE #85527: punji stick invisible on perspeciteve
## Issue Summary

punji stick cant be seen in  Walening tsetmerge in certain perspectives -such as horizontal doors

## Reproduction:
image

Make punji stick in horizontal airlock
It is invisible

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
EDIT: Ok, I've found a way to make the bottom half of left/right-facing airlocks transparent in a way that it doesn't block clicks on items and mobs standing between the two ends.

immagine

Also I've shifted the icons for punji and woodspikes two pixels up. EDIT: Also taking care of some potential hard dels with stasis beds and their visual overlays.

## Why It's Good For The Game
This will fix #85527.

## Changelog

:cl:
qol: Gave some transparency to the bottom half of left/right-facing doors when open. They shouldn't block clicks on mobs and items now.
/:cl:

### CODE REVIEW

**Comment by MrMelbert** [comment]
Is this the "right" fix? Shouldn't the transparent frill effect be applied here?

**Comment by Ghommie** [comment]
> Is this the "right" fix? Shouldn't the transparent frill effect be applied here?

Oh, tell me about it, how does it work?

**Comment by LemonInTheDark** [comment]
Would like screenshots of this, also I do not think it makes sense applied to (at least) shutters.

> Also to clarify I'm talking about the effect where the top 8 pixels of a wall turn invisible when you stand by them

This is done by drawing the "frill" of the window to a unique plane that gets masked by another plane with the client's outline on it.
It may not have the desired layering in this case since we are unable to cheaply divorce layer from effect grouping (shakes fist at sky)

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -571,8 +571,12 @@
 			frame_state = AIRLOCK_FRAME_OPEN
 			// If we're open we layer the bit below us "above" any mobs so they can walk through
 			if(!short_rendering)
-				. += mutable_appearance(icon, "open_bottom", ABOVE_MOB_LAYER, appearance_flags = KEEP_APART)
-				. += emissive_blocker(icon, "open_bottom", src, ABOVE_MOB_LAYER)
+				var/mutable_appearance/bottom = mutable_appearance(icon, "open_bottom", ABOVE_MOB_LAYER, appearance_flags = KEEP_APART)
+				bottom.alpha = 185
+				. += bottom
+				bottom = emissive_blocker(icon, "open_bottom", src, ABOVE_MOB_LAYER)
```
Comment (LemonInTheDark) [inline code/game/machinery/doors/airlock.dm:577]: these procs have alpha args, you should use them. Especially for emissive blockers if memory serves what you're doing here is gonna fuck with the applied color matrix or someodd.

**Pull request closed without merging**